conf/logback.xml

```diff
@@ -9,7 +9,7 @@
     </appender>
 
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-        <file>${application.home}/logs/donut.log</file>
```

Comment:
Sometimes, not every time,  ${application.home} which I thought was a default Logback variable, seemed not to be set correctly, and produced an additional "${application.home} _not_set" directory with the log files in my local environment.